FluidNC/src/WebUI/WifiConfig.cpp

```diff
@@ -442,6 +499,17 @@ namespace WebUI {
                     log_stream(out, "Gateway: " << IP_string(IPAddress(WiFi.softAPIP())));
                     log_stream(out, "Mask: 255.255.255.0");
 
+#ifdef IDFBUILD
```

Comment:
Could this be based on the IDF version instead of IDFBUILD?

Reply:
No, i tried that. Arduino also compiles using idf at some point, so it defines the same symbol during compilation. I can check the version as well, but must add a -D to cmakelists to be able to differentiate between the frameworks.

Reply:
Changed what I could. Commit 84ae213bcd6cdd2b1fc4f320d1ed984b3539efa3